frontend/src/components/ShootWorkers/VolumeType.vue

```diff
@@ -88,16 +132,51 @@ export default {
         return 'This volume type may not be supported by your worker'
       }
       return undefined
+    },
+    isAWS () {
+      const cloudProfile = this.cloudProfileByName(this.cloudProfileName)
+      return get(cloudProfile, 'metadata.cloudProviderKind') === 'aws'
+    },
+    workerIops: {
+      get () {
+        return this.internalIops
+      },
+      set (value) {
+        this.internalIops = value
+        if (value > 0) {
+          if (!this.worker.providerConfig) {
+            this.worker.providerConfig = getWorkerProviderConfig('aws')
+          }
+          set(this.worker.providerConfig, 'volume.iops', parseInt(value))
+        } else {
+          unset(this.worker.providerConfig, 'volume.iops')
+        }
+      }
+    },
+    maxIops () {
```

Comment:
hardcoded volume types?

Reply:
Yes according to the aws specifications, this values should not change, there is no point in making them configurable and we can also not read them from anywhere, right? @rfranzke

Reply:
There are some limits imposed by AWS, https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/ebs-volume-types.html, but now taking a look at it it seems that they have increased them.

I think for `gp2` its `[100,16000]` meanwhile, and for `io1` its `[100,64000]`.